lib/run/index.js

```diff
@@ -156,6 +155,9 @@ module.exports = function (options, callback) {
         // to store the exported content from reporters
         emitter.exports = [];
 
+        // expose the runner object for reporter and programmatic use
+        emitter.runner = runner;
+
```

Comment:
Sorry if it's a noob question, but why do we need to expose the `runner` object. AFAIK, we only need the `run` object (which we're exposing in `start` event) for programmatic access, right?

Reply:
Yes. I've exposed it in case right before start (at the construction point of the reporter), we need to configure the runner options or do some function wrapping shenanigans. 😛